libchisel/src/gc.rs

```diff
@@ -0,0 +1,228 @@
+extern crate parity_wasm;
+extern crate wasm_gc;
+
+use super::ModuleTranslator;
+use parity_wasm::elements::*;
+
+pub struct WasmGC(wasm_gc::Config);
+
+impl Default for WasmGC {
+    fn default() -> WasmGC {
+        WasmGC(wasm_gc::Config::new())
```

Comment:
Why cannot we use the default trait here?

Reply:
Does `Config` implement or derive `Default`? I very rarely see `Default` used in the wild. Implementing `new` is much cleaner most of the time.

Reply:
Checked after I commented and it didn't.